lib/anki_viewer_web/channels/sync_channel.ex

```diff
@@ -21,16 +21,23 @@ defmodule AnkiViewerWeb.SyncChannel do
 
     push(socket, "sync:msg", %{msg: "updated collection"})
 
-    Repo.delete_all(Note)
+    new_cards_data = AnkiViewer.cards_data!()
 
-    notes_data = AnkiViewer.notes_data!()
+    current_cards_data = Repo.all(Card)
 
-    for {n, i} <- Enum.with_index(notes_data) do
-      push(socket, "sync:msg", %{msg: "updating note #{i}/#{length(notes_data)}"})
-      Note.insert!(n)
-    end
+    filtered_new_cards_data =
```

Comment:
Add a TODO comment here saying that we also need to delete and update cards here

Reply:
This can be fixed when #23 is addressed